portfolio/src/main/webapp/script.js

```diff
@@ -113,15 +113,34 @@ async function getComments() {
   console.log('Handling the response.');
   const text = await response.json();
   console.log('/data: ' + text);
-  const commentsContainer = document.getElementById('previous_comments-container');
-  // TODO: display comments in DOM
+  const commentsContainer = document.getElementById('previous-comments');
+  text.forEach(entry => {
+    commentsContainer.appendChild(createListElement(entry));
+  });
 }
    
 /**
- * Creates an <li> element containing text. 
+ * Creates an <li> element containing comment entries. 
  */
-function createListElement(text) {
+function createListElement(entry) {
```

Comment:
Consider choosing a more specific name for this function.

In the code above:

```javascript
text.forEach(entry => {
  commentsContainer.appendChild(createListElement(entry));
});
```

it's not immediately clear what `createListElement` is doing, a reader would have to jump to the function to see what's going on.

Something like `createListElementForComment` makes it immediately clear, and will also help with potential confusion if you end up creating more list element helpers down the road.